Revit_Core_Engine/Query/DrawingArea.cs

```diff
@@ -37,7 +37,32 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
-        [Description("Compute the outline of Title Block drawing area.")]
+        [Description("Returns the drawing area outline of the Title Block instance.")]
+        [Input("titleBlock", "Title Block to get the drawing area outline from.")]
+        [Output("outline", "The Title Block's drawing area.")]
+        public static Outline DrawingArea(this FamilyInstance titleBlock)
+        {
+            if (titleBlock == null || !((BuiltInCategory)titleBlock.Category.Id.IntegerValue == Autodesk.Revit.DB.BuiltInCategory.OST_TitleBlocks))
+            {
+                BH.Engine.Base.Compute.RecordWarning($"Title block cannot be null and has to be of Title Block category.");
+                return null;
+            }
+
+            Outline drawingArea = titleBlock.Symbol.DrawingArea();
+            Transform transform = titleBlock.GetTotalTransform();
+
+            if (!transform.IsIdentity)
+            {
+                drawingArea.MaximumPoint += transform.Origin;
+                drawingArea.MinimumPoint += transform.Origin;
```

Comment:
I think we should use all data in the Transform, even if it's only a movement like in this case.
```suggestion
                drawingArea.MaximumPoint = transform.OfPoint(drawingArea.MaximumPoint);
                drawingArea.MinimumPoint = transform.OfPoint(drawingArea.MinimumPoint);
```